src/components/Icon/icons/Attach.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+export default function Attach(props: any) {
+  return (
+    <svg viewBox="0 0 320 320" xmlns="http://www.w3.org/2000/svg" xmlnsXlink="http://www.w3.org/1999/xlink" aria-hidden="true" {...props}>
```

Comment:
these might need height and width attributes to match the viewBox bounds for proper scaling, both 320 in this case (although some of our icons have different dimensions right now)

i think we should also delete the `xmlnsXlink` attributes since we're not using `<anchor>` links in SVG:
https://en.wikipedia.org/wiki/XLink

Reply:
Agreed. 

I think svgo is going to remove xmlnsXlink, so we don't have to worry about that, unless for some reason we can't use svgo (it may be messing up some of the icons). But we do need to make sure that height and width are set as well as viewBox (our svgs don't always have viewBox).